### PR TITLE
updating Stargate description 

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
only updating copy for the # Stargate section

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,8 +23,7 @@ by adding plugin support for new APIs, data types, and access methods.
 We created Stargate because we got tired of using different databases and different APIs depending on the work that we were trying to get done.
 With "read the manual" fatigue and lengthy selection processes wearing on us every time we started a new project, we thought - *why not create a framework that can serve many APIs for a range of workloads?*
 
-This project enables customization of all aspects of data access and has modules for authentication, APIs, request handling / routing, and persistence backends.
-The current form is specific to the Apache Cassandra (C*) backend but there's no bounds to the databases or APIs that this framework can support.
+This project enables customization of all aspects of data access and has modules for authentication, APIs, request handling / routing, and persistence backends.  The current form is specific to the Apache Cassandra (C*) backend but there's no bounds to the databases or APIs that this framework can support.
```
Comment (tatu-at-datastax) [inline README.md:26]: This project -> Stargate?

Comment (tatu-at-datastax) [reply README.md:26]: also, I would probably remove "no bounds to the databases". It is simply not true; system is heavily dependant on Cassandra being the backend. One can "fake" other databases of course (like Mongo or DynamoDB) but that's bit different.

Comment (jeffreyscarpenter) [reply README.md:26]: @pieterhumphrey WDYT?

Comment (tatu-at-datastax) [reply README.md:26]: Maybe "The current form..." could become:

"Backend implementations currently exists for multiple Apache Cassandra (C*) versions although the framework itself is not strongly tied to Cassandra."

I am bit too tired to come up with a better explanation.
Alternatively that second sentence could simply be removed.

**Comment by jeffreyscarpenter** [comment]
Having some issues with editing this branch, will close and redo

File: README.md (hunk 1 above)
Comment (pieterhumphrey) [reply README.md:26]: remove it IMHO, AFAIK it is strongly tied to Cassandra and non-Cassandra DBs are a non-goal of the project ATM

**Pull request closed without merging**